versions/latest/GAME_MASTER.json

```diff
@@ -41164,6 +41344,11 @@
       "kmBuddyDistance": 3.0,
       "buddySize": "BUDDY_BIG",
       "modelHeight": 1.55,
+      "evolutionBranch": [{
+        "evolution": "DUSKNOIR",
+        "evolutionItemRequirement": 1106,
```

Comment:
Looks like the Sinnoh Stone ID wasn't part of the protobuf parser, that should probably be fixed before this gets merged.

Reply:
"ignore" I have nothing to say, it's okay just to see if it goes https://discord.gg/8a4XnCz